src/pages/ImportData.js

```diff
@@ -121,7 +125,10 @@ const ImportData = () => {
       Utils.withBusyState(setIsImporting),
       withErrorReporting('Error loading templates')
```

Comment:
This error message isn't quite right anymore. I think it should be `Error loading initial data` or something?